app/js/components/manageVisit.jsx

```diff
@@ -0,0 +1,274 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React from 'react';
+import toastr from 'toastr';
+import moment from 'moment';
+import apiCall from '../utilities/apiHelper';
+import DateTimeField from 'react-bootstrap-datetimepicker';
+import { Button, Form, FormGroup, Label, Input, FormText, Table, ListGroup, ListGroupItem } from 'reactstrap';
+import { withRouter } from 'react-router'
+
+const getSuggestionValue = suggestion => suggestion.name;
+
+class ManageVisit extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      uuid: this.props.params.patentId,
+      visituuid: this.props.params.visitId,
+      display: '',
+      location: [],
+      locationName: '',
+      patientName: '',
+      format: "YYYY-MM-DD HH:mm:ss",
+      inputFormat: "YYYY-MM-DD HH:mm:ss",
+      mode: "date",
+      startDatetime: '',
+      finalStartTime: '',
+      finalStopDatetime: '',
+      stopDatetime: '',
+      visitType: [],
+      visitTypeName: '',
+      voided: false,
+      encounters: [],
+      patients: [],
+      searchTerm: 'm',
+
+
+
+    };
+    this.handleEncounterClick = this.handleEncounterClick.bind(this);
+    this.handleTimeChange = this.handleTimeChange.bind(this);
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSave = this.handleSave.bind(this);
+    this.handleDelete = this.handleDelete.bind(this);
+    this.goHome = this.goHome.bind(this);
+  }
+
+
+  componentWillReceiveProps(nextProps) {
+    if (this.state.uuid !== this.props.params.patentId) {
+      this.setState({
+        uuid: this.props.params.patentId
+      })
+
+    }
+  }
+
+  componentDidMount() {
+    apiCall(null, 'get', `/visit/${this.state.visituuid}`)
+      .then((res) => {
+        const { display, encounters, location, patient, startDatetime: time, stopDatetime: time2, visitType, voided } = res
+        const locationName = location.display
+        const patientName = patient.display
+        const visitTypeName = visitType.display
+        const startDatetime = moment(time).format('YYYY-MM-DD HH:mm:ss')
+        const finalStartTime = moment(startDatetime).format('YYYY-MM-DDTHH:mm:ss.000Z')
```

Comment:
Is this format intentionally different from the one above?

Reply:
@dkayiwa the date-time picker uses the 'YYYY-MM-DD HH:mm:ss' format.. that is why i converted it to that,then once one picks a date and time it is converted to the one used by backend,'YYYY-MM-DDTHH:mm:ss.000Z' format so that we can post data to the db